src/core/maps/area_occupancy_estimator.h

```diff
@@ -109,7 +129,18 @@ class AreaOccupancyEstimator : public CellOccupancyEstimator {
     }
 
     double corner_x = 0, corner_y = 0, area = 0;
+
     double chunk_is_triangle = inters[0].is_horiz() ^ inters[1].is_horiz();
+    /* inters[0].is_horis | inters[1].is_horiz | XOR (^)
```

Comment:
consider a shorter version of clarification: "A chunk is a triangle if a beam crosses orthogonal sides of a cell".
Btw, the comment is private.